params/network_upgrades.go

```diff
@@ -23,6 +27,11 @@ var (
 		SubnetEVMTimestamp: utils.NewUint64(0),
 		// DUpgradeTimestamp: utils.NewUint64(0), // TODO: Uncomment and set this to the correct value
 	}
+
+	UnitTestNetworkUpgrades = MandatoryNetworkUpgrades{
```

Comment:
Instead of depending on a `defaultActivationTime` that we need to be aware of throughout the tests, can we change the behavior of overriding network upgrades to ONLY override the mandatory network upgrades for a predetermined set of networkIDs so that if our tests use a different networkID intentionally it is not overridden?

Reply:
done in :https://github.com/ava-labs/subnet-evm/pull/839